roles/openshift_master/defaults/main.yml

```diff
@@ -29,4 +29,4 @@ os_firewall_deny:
 - service: former etcd peer port
   port: 7001/tcp
 
-openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag | default(openshift.docker.openshift_image_tag | default(''))) }}"
+openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
These default filters should be nested, otherwise there will be an error for any variable used in a default filter that is undefined.